x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_management/hooks/use_create_analytics_form/state.ts

```diff
@@ -9,8 +9,11 @@ import { checkPermission } from '../../../../../privilege/check_privilege';
 
 import { DataFrameAnalyticsId, DataFrameAnalyticsConfig } from '../../../../common';
 
-const OUTLIER_DETECTION_DEFAULT_MODEL_MEMORY_LIMIT = '50mb';
-const REGRESSION_DEFAULT_MODEL_MEMORY_LIMIT = '100mb';
+export enum DEFAULT_MODEL_MEMORY_LIMIT {
+  regression = '100mb',
+  // eslint-disable-next-line @typescript-eslint/camelcase
+  outlier_detection = '50mb',
```

Comment:
Is this used? When I start in the wizard, and set the type to Outlier detection, the default stays at 100mb.

Reply:
Yes this is used as the default value placeholder for when jobType is outlier detection.